python/paddle/v2/framework/tests/test_sigmoid_op.py

```diff
@@ -12,5 +12,8 @@ def setUp(self):
         self.Y = 1 / (1 + np.exp(-self.X))
 
 
+#class TestSigmoidGradOp(unittest.TestCase):
```

Comment:
@dzhwinter Thanks, I'll add unit test after the unit test is ready for the backward op.